### PR TITLE
Translated 1,328 word to tagalog

### PR DESCRIPTION
**Pull request by jjrang11**
Translated the following files to Tagalog
progit2/book/04-git-server/1-git-server.asc
progit2/book/04-git-server/sections/smart-http.asc
progit2/book/04-git-server/sections/hosted.asc

### CODE REVIEW

File: book/04-git-server/1-git-server.asc [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-First, you choose which protocols you want your server to communicate with.
-The first section of this chapter will cover the available protocols and the pros and cons of each.
-The next sections will explain some typical setups using those protocols and how to get your server running with them.
-Last, we'll go over a few hosted options, if you don't mind hosting your code on someone else's server and don't want to go through the hassle of setting up and maintaining your own server.
+Ang pagpapatakbo ng isang Git Server ay tuwiran.
+Una, pipili ka kung anong mga protokol ang gusto mo gamitin ng iyong server sa pag-usap.
+Sinasaklaw ng unang bahagi ng kabanatang ito ang mga magagamit na mga protokol at ang mga kalamangan at kahinaan ng bawat isa.
+Ang mga susunod na mga bahagi ay magpapaliwanag sa ilang tipikal na mga set up na gamit ang mga protokol na iyon at kung papaano mapapatakbo ang iyong server gamit ang mga iyon.
+Sa Wakas, Tatalakayin natin ang ilang mga naka-host na opsyon, kapag wala kang pakialam na i-host ang iyong code sa server ng ibang tao at ayaw mong dumaan sa abala ng pag-set up at pagpapanatili ng sariling server.
```
Comment (heywens) [inline book/04-git-server/1-git-server.asc]: kindly change ``Wakas`` at ``Tatalakayin`` to lower cases

File: book/04-git-server/1-git-server.asc [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 
-If you have no interest in running your own server, you can skip to the last section of the chapter to see some options for setting up a hosted account and then move on to the next chapter, where we discuss the various ins and outs of working in a distributed source control environment.
+Kapag hindi ka interesado sa pagpapatakbo ng sarili mong server, maaari kang lumaktaw sa huling bahagi ng kabanata upang makita ang ilang mga opsyon sa pag-set up ng isang naka-host na akawnt at pagkatapos ay lumipat sa susunod na kabantan, kung saan ay tatalakayin natin ang iba't ibang ins at outs sa pagtatrabaho sa isang distributed source control environment.
```
Comment (heywens) [inline book/04-git-server/1-git-server.asc]: ``kabantan`` to ``kabanata``

File: book/04-git-server/sections/hosted.asc [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-Doing so offers a number of advantages: a hosting site is generally quick to set up and easy to start projects on, and no server maintenance or monitoring is involved.
-Even if you set up and run your own server internally, you may still want to use a public hosting site for your open source code – it's generally easier for the open source community to find and help you with.
+Kung hindi mo nais pagdaanan ang lahat ng trabaho kasali sa pag-set up ng iyong sariling Git server,  mayroon kang iilang opsyon para i-host ang iyong mga proyekto sa Git sa isang dedikadong panlabas na hosting site.
+Ang paggawa nito ay nag-aalok ng ilang mga kalamangan: ang isang hosting site ay sa pangkalahatan ay daling i-set up at madaling magsimula ng mga proyekto, at walang maintenance sa server o kailangan ng pagsubaybay.
+Kahit na magset-up ka at mapaktbo ang iyong server internally, maaari mo parin gusto gumamit ng isang pampublikong hosting site para sa iyong open source na mga code -- sa pangkalahatan mas madali para sa open source na komunidad na hanapin at tulungan ka.
```
Comment (heywens) [inline book/04-git-server/sections/hosted.asc]: * ``mapaktbo`` to ``magpatakbo``
* ``parin`` > add space character

File: book/04-git-server/sections/hosted.asc [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
 
-These days, you have a huge number of hosting options to choose from, each with different advantages and disadvantages.
-To see an up-to-date list, check out the GitHosting page on the main Git wiki at https://git.wiki.kernel.org/index.php/GitHosting[]
+Sa mga araw na ito, ikaw ay mayroong isang malaking bilang ng mga opsyon sa hosting na pagpipilian, bawat isa ay may mga kalamangan at kawalan.
+Upang makita ang isang up-to-date na listahan, tingnan ang pahina ng GitHosting sa pangunahing wiki sa https://git.wiki.kernel.org/index.php/GitHosting[]
 
-We'll cover using GitHub in detail in <<_github#_github>>, as it is the largest Git host out there and you may need to interact with projects hosted on it in any case, but there are dozens more to choose from should you not want to set up your own Git server.
+Tatalakayin namin ng detalyado ang paggamit ng Github sa <<_github#_github>>, dahil ito ang pinakamalaking Git host doon at maaaring kailangan mong makipag-ugnayan sa mga proyekto na naka-host roon sa anumang kaso, ngunit mayroong dose-dosenang higit pa na pagpipilian kung hindi mo nais mag-set up nga iyong sariling Git server.
```
Comment (heywens) [inline book/04-git-server/sections/hosted.asc]: ``nga` to ``ng`

File: book/04-git-server/sections/smart-http.asc [hunk 5]
```diff
@@ -34,9 +34,9 @@ SetEnv GIT_HTTP_EXPORT_ALL
 ScriptAlias /git/ /usr/lib/git-core/git-http-backend/
 ----
 
-If you leave out `GIT_HTTP_EXPORT_ALL` environment variable, then Git will only serve to unauthenticated clients the repositories with the `git-daemon-export-ok` file in them, just like the Git daemon did.
+Kung hindi mo isasali ang `GIT_HTTP_EXPORT_ALL` na variable ng environment, pagkatapos ay ihahain lamang ng Git sa mga hindi awtentik na mga kliyente ang mga repositoryo na may `git-daemon-export-ok` na file sa loob nila, pareho sa ginawa ng Git daemon.
 
-Finally you'll want to tell Apache to allow requests to `git-http-backend` and make writes be authenticated somehow, possibly with an Auth block like this:
+Sa wakas nais mo sabihan ang Apache na pahintulutan ang mga request patungo sa `git-http-backend` at maghanap ng paraan na gawing awtentik ang mga pagsulat, posible sa isang block ng Auth kagaya nito:
```
Comment (heywens) [inline book/04-git-server/sections/smart-http.asc]: ``mo`` to ``mong``?

File: book/04-git-server/sections/smart-http.asc [hunk 6]
```diff
@@ -49,24 +49,24 @@ Finally you'll want to tell Apache to allow requests to `git-http-backend` and m
...
 
 [source,console]
 ----
 $ htpasswd -c /srv/git/.htpasswd schacon
 ----
 
-There are tons of ways to have Apache authenticate users, you'll have to choose and implement one of them.
-This is just the simplest example we could come up with.
-You'll also almost certainly want to set this up over SSL so all this data is encrypted.
+Mayroong toneladang paraan na kung saan masagawang awtenik ng Apache ang mga user, kailangan mo pumili ng isa at ipatupad ito.
```
Comment (heywens) [inline book/04-git-server/sections/smart-http.asc]: ``mo`` to ``mong``

File: book/04-git-server/sections/smart-http.asc [hunk 7]
```diff
@@ -49,24 +49,24 @@ Finally you'll want to tell Apache to allow requests to `git-http-backend` and m
...
+Ito ang pinakasimpleng halimbawa na maaari nating gawin.
+Makikita mo rin ang tiyak na nais na i-set up ito gamit ang SSL upang lahat ng data nito ay encrypted.
 
-We don't want to go too far down the rabbit hole of Apache configuration specifics, since you could well be using a different server or have different authentication needs.
-The idea is that Git comes with a CGI called `git-http-backend` that when invoked will do all the negotiation to send and receive data over HTTP.
-It does not implement any authentication itself, but that can easily be controlled at the layer of the web server that invokes it.
-You can do this with nearly any CGI-capable web server, so go with the one that you know best.
+Hindi natin nais na talakayin ng malalim ang mga tiyak na pagsasaayos ng Apache, dahil maaaring gumagamit ka ng ibang server o ibang pangangailangan ng pagpapatunay.
+Ang idea ay ang Git ay may kasamang isang CGI na ang tawag ay `git-http-backend` na kung tinawag ay isasagawa ang lahat ng pag-aayos upang maipadala at matanggap ang mga datos gamit ang HTTP.
+Walang itong pagpapatunay na isinasagawa, ngunit madali ito kontrolin sa layer ng web server na tumatawag nito.
```
Comment (heywens) [inline book/04-git-server/sections/smart-http.asc]: ``Walang`` to ``Wala``

**Comment by heywens** [comment]
Hi @jjrang11, I have suggested minor changes on your PR. 
Kindly check. Thanks.

**Comment by jjrang11** [comment]
hello @heywens thank you for proofreading my work! I have updated the changes as you suggested

File: book/04-git-server/1-git-server.asc [hunk 8]
```diff
@@ -1,24 +1,24 @@
...
-== Git on the Server
+== Git sa Server
 
 (((serving repositories)))
-At this point, you should be able to do most of the day-to-day tasks for which you'll be using Git.
-However, in order to do any collaboration in Git, you'll need to have a remote Git repository.
-Although you can technically push changes to and pull changes from individuals' repositories, doing so is discouraged because you can fairly easily confuse what they're working on if you're not careful.
-Furthermore, you want your collaborators to be able to access the repository even if your computer is offline -- having a more reliable common repository is often useful.
-Therefore, the preferred method for collaborating with someone is to set up an intermediate repository that you both have access to, and push to and pull from that.
+Sa puntong ito, maari mo na dapat gawin ang mga pang-araw-araw na gawain kung saan gagamitin mo ang Git.
```
Comment (josteinfrancois) [inline book/04-git-server/1-git-server.asc]: Please change "maari" to "maaari"

File: book/04-git-server/1-git-server.asc [hunk 9]
```diff
@@ -1,24 +1,24 @@
...
+Gayunpaman, upang makagawa ng anumang kolaborasyon sa Git, kakailanganin mong magkaroon ng remote na respositoryo ng Git.
+Kahit na technically maari mong i-push ang iyong mga pagbabago at i-push ang mga pagbabago mula sa mga repositoryo ng mga indibidwal, ang paggawa nito ay hindi hinihikayat dahil madali malito sa kanilang pinagtrabuhan kapag hindi ka maingat.
```
Comment (josteinfrancois) [inline book/04-git-server/1-git-server.asc]: Here also, please change "maari" to "maaari"

**Comment by jjrang11** [comment]
hello @josteinfrancois thank you for spotting that. I've updated it already

**Pull request merged**